loadimpact/k6-docs
# Feature/repo maintenance

**Describe changes**
Repo maintenance, roughly the same as recent PRs in main and blog repos. 

- Removed unused deps, stale stories
- Updated everything
- Set up dependabot (set updating interval `monthly` this time)
- Made sure images are on their places (had to disable `webp` srcset generation as well, there is a bug in one of the gatsby plugins, will raise an issue in gatsby repo)
- Set up husky and lint staged
- Tweaked esling and prettier configs to match the other repos set up

**Steps to test**
- Pull the changes
- Run the project
- See that everything is working as expected

After that we go the usual way: deploy to `staging`, then merge into `develop`
